doc/topics/cloud/windows.rst

```diff
@@ -9,6 +9,13 @@ Windows images.
 
 Requirements
 ============
+
+.. note::
+   Support ``winexe`` and ``impacket`` has been deprecated and will be removed in
+   Fluorine. These dependencies are replaced by `pypsexec` and `smbprotocol`
```

Comment:
The back-ticks around pypsexec and smbprotocol should be double back-ticks for these to format properly in the docs.